### ISSUE TITLE
prompt is doubled (line messed up)

### ISSUE THREAD

**Issue by shula** [mridgers/clink#429]
**edit**
this problem stems from CLink vs. ConEmu integration.  Clink by itself works perfectly.
see my comment below

-------- original issue --------

my prompt is doubled.  it's probably my previous settings, but i don't know how to clean them.

- clink 4.8
- windows 7
- previously had older version of clink + that multitab console app (i can't remember its name)

steps:

1. run CMD
2. on the prompt, C:\users\user>  press ENTER
3. the prompt from now and on is **C:\users\user>C:\users\user>** dir

1. ~~i suspect it's because i have a proportional font (arial), and not the customary fixed-width font ~~ Ruled out. now using courier new, still same.
2. i clicked "reset settings", restarted clink, but still same behaviur
3. i tried to change the prompt using "prompt" command, to $p$g  -- nothing changed.

did you experience such behaviour?
i'm still investigating the settings, and will update this thread,

**Comment by Equinox** [comment]
I'm experiencing this too:

I can repo when using Clink in [Hyper](https://hyper.is/), but not when using Clink in just cmd.exe.

It's quite bad in use with Hyper too, as just zooming in can make the text duplicate (i.e "C:\users\user>" becomes "C:\users\user>C:\users\user>")

**Comment by shula** [comment]
I realized these errors stem from CLINK vs. ConEmu integration.

 - it depends on the order of execution, order of install the programs, 
 - it depends on which parent program was launching CMD.EXE:

When executing CMD from some programs, conEmu is injected, and the console is buggy 
(when clink & conemu running together). 

Example of parent programs that mess up with cLink-ConEmu duo::

 - Clavier+
 - Total commander

parent-programs that has **no** bugs observed when running cmd.exe:

 - the "Run..." dialog (i.e. Win+R > "cmd", press Enter).
 - from a python program:  os.system('cmd')


see also:  
https://github.com/mridgers/clink/issues/339

After some tests I decided to uninstall conEmu (I don't need it), and no bugs found.
